Bottom: 8cd19786528f6ad2f3f891d4d36554ee65a2fa2b
Top:    3b45f348511d2eedefff255f39c22c8a16af9665
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-13 17:22:16 -0400

Refresh of mpc5200-ac97

---

diff --git a/sound/soc/fsl/mpc5200_psc_ac97.c b/sound/soc/fsl/mpc5200_psc_ac97.c
index 340739c..385f697 100644
--- a/sound/soc/fsl/mpc5200_psc_ac97.c
+++ b/sound/soc/fsl/mpc5200_psc_ac97.c
@@ -9,8 +9,6 @@
  * published by the Free Software Foundation.
  */
 
-#define DEBUG
-
 #include <linux/init.h>
 #include <linux/module.h>
 #include <linux/interrupt.h>
